### PR TITLE
Add even more analytics

### PR DESCRIPTION
**Pull request by mommothazaz123**
### Summary
Adds analytics for:
- commands called per day
- characters imported per day
- number of active users last day/week/month
- number of active guilds last day/week/month

This script will run daily at 3am CST (1am PST) on AWS Lambda. It grabs the URL to connect to MongoDB through Secrets Manager.

#### PR Type

- [ ] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [x] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

File: scripts/lambda/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Lambda Scripts
```
Comment (zmckinnon) [inline scripts/lambda/README.md]: We could probably just put it in a stand alone repository.  That will make it easier to setup CI/CD too.

Comment (mommothazaz123) [reply scripts/lambda/README.md]: That makes sense - the environment is independent of the rest of the project, anyway. I'll close this after reviews and make `avrae/lambda-scripts` instead.

File: scripts/lambda/daily_analytics.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    # setup
+    out = {"timestamp": datetime.datetime.now()}
+    to_date = dict()
+
+    # --- calculations ---
+
+    # -- deltas --
+    # most popular commands today
+    # fixme: how does this work in a columnal data store?
+    # out['command_activity'], to_date['command_activity'] = calculate_command_activity(last_to_date)
```
Comment (mommothazaz123) [inline scripts/lambda/daily_analytics.py]: The code to calculate these is above (`calculate_command_activity()`), but it outputs a dict that looks like:
```
{
    "roll": 10321,
    "check": 7130,
    ... (~70 entries)
}
```
that just doesn't work well for a columnal data store.

**Comment by mommothazaz123** [comment]
https://github.com/avrae/lambda

**Pull request closed without merging**